From e7b04b5b5bc821e7441b1dae50185546201cca63 Mon Sep 17 00:00:00 2001
From: Mark Asselstine <mark.asselstine@windriver.com>
Date: Wed, 19 Dec 2012 21:18:03 -0800
Subject: [PATCH 03/16] grsec: powerpc: pass const to kvm_arch_init

In commit [GRSecurity 2.9.1 -- 201207080925] the argument
for kvm_arch_init is changed to use a const modifier, reflect
that change here.

Signed-off-by: Mark Asselstine <mark.asselstine@windriver.com>
---
 arch/powerpc/kvm/powerpc.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/powerpc/kvm/powerpc.c b/arch/powerpc/kvm/powerpc.c
index 6316ee3..7ee24e2 100644
--- a/arch/powerpc/kvm/powerpc.c
+++ b/arch/powerpc/kvm/powerpc.c
@@ -1121,7 +1121,7 @@ void kvmppc_init_lpid(unsigned long nr_lpids_param)
 	memset(lpid_inuse, 0, sizeof(lpid_inuse));
 }
 
-int kvm_arch_init(void *opaque)
+int kvm_arch_init(const void *opaque)
 {
 	return 0;
 }
-- 
1.7.9.5

